pkg/ccl/multiregionccl/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ go_library(
 
 go_test(
     name = "multiregionccl_test",
+    size = "enormous",
```

Comment:
this should probably just be medium or large.

Reply:
I ran the test on my machine and it didn't conclude within 5m 🤷 https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/1553172016/How+to+ensure+your+tests+can+run+in+the+Bazel+sandbox#Test-size

Reply:
random driveby comment -- can the test runner be updated to say "Test timed out after X duration" rather than "Test timed out at time X"? i was just looking at these logs https://teamcity.cockroachdb.com/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=2810604&_focus=39520

Reply:
what a big boi

Reply:
> random driveby comment -- can the test runner be updated to say "Test timed out after X duration" rather than "Test timed out at time X"?

Probably, but that's not happening immediately -- we haven't made any adjustments to the test runner at all so far, although that's on the roadmap.